ops/dhparams.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN DH PARAMETERS-----
+MIICCAKCAgEAuknRHBPKrwxTsCO38bmvH5lxBXRDLuNTKuzmonKpDPmNbEBmSdMK
+F1Onv7rcSZKiqSCNp1lcscFNMOFbCAVbWMVm7qfSDNMT5EITjXgDWKN81jIpcCzv
+wHHP0JqIK7KEG6CYDOekoKbUSA50UrdMvyJyh1eEVXbwdsJ3TQmShOxAsm5OFNgc
+jet6O7VJ+fLtaKj2n0YKN0y//x0QNtIosQGPlj/RFa4HqRCgacRxh9BOTm9M78KW
+hIbolEgrjy4hCB3ikAwse5X48Fz2khN6s+MLuOWOBq06tKzthsUTpEuO9NPabs82
+a+plYExwzWTdHOdSkZymNGfv76/FN+IVw8UD6wiQNyqRZcggvRA7udWTP9SxHiSR
+o2gaJTd5v+dSIKn8NEzel7ggvZpt1X+K6kFtReb7aoymjLdfXxgZ0bLHUBXDeIhG
+f74ChO2ACKCVcWqfTA79abnvz1tskG8BYpIX0Tgx+j3xo/GWYCMzWOqXBD4NtsMP
+zJux91Hg765AeVmtZ50Ez4VggcaiEbao2JN55Nsu9ffzCPmu8skiEzzosCD1RM+D
+PHcVk35AmrPrbZg8CJ364omS17APXPaXL4QZ0YXZgT0uwHjv/VJxH2BXxWuw7ka4
++8D/6lgWEA51p21+dmER5XhofD/ZRkRSr/shA6iT/9R2B8lgph4KoZsCAQI=
+-----END DH PARAMETERS-----
```

Comment:
This file contains the generator and prime parameters for Diffie-Hellman that is used by nginx. Although using Diffie-Hellman key is optional while configuring letsencrypt certificates with nginx but it is primarily used as a method of exchanging cryptography keys securely for use in the letsencrypt certificate.

It takes about 40 seconds to generate a 2048 bit set and it take and eternity to generate a 4096 bit set.

I'm open to having this file file generated inside the docker container for the 2048 bit and not storing in the repo. Either way, it is safe to have this file in the public. Even certbot has this file publicly viewable: https://github.com/certbot/certbot/blob/master/certbot/ssl-dhparams.pem

Reply:
Gotcha I get it now 👌Thanks for explaining it!